src/EventListener/LocaleListener.php

```diff
@@ -32,7 +32,7 @@
 class LocaleListener implements EventSubscriberInterface
 {
     /**
-     * @var string Default framework locale
+     * @var string|null Default framework locale
```

Comment:
All good in my opinion, my only doubt is about whether defaultLocal could be nullable or not.

Reply:
@kamilmusial by looking at your implementation, this can never be null